src/index.tsx

```diff
@@ -130,6 +130,7 @@ export interface Props {
   size?: Size;
   resizeGrid?: Grid;
   bounds?: string | Element;
+  resizeBounds?: string | Element;
```

Comment:
I want to minimize the number of props as much as possible.
Can the issue be addressed by expanding the `bounds` as follows?

``` TypeScript
bounds?: (string | Element) | { type: 'resize': bounds: string | Element } |   { type: 'drag': bounds: string | Element } 
```

Reply:
So there's times where I want to pass both bounds to the object 

This means bounds then has to either become an array with both bounds types in it, or an object with keys of resize and drag, which do you prefer?

Reply:
I see. Please wait a moment.

Reply:
OK great, I'll try and get round to this in the next few days, thanks